engine/Scripts/lib_build.py

```diff
@@ -137,22 +141,27 @@ def set_params():
 def set_env_verb():
     global platform
     global flutter_root_path
-    flutter_root_path = os.getenv('FLUTTER_ROOT_PATH', 'null')
-    if flutter_root_path == 'null':
-        os.environ["FLUTTER_ROOT_PATH"] = os.path.join(engine_path, "engine","src")
-        flutter_root_path = os.getenv('FLUTTER_ROOT_PATH')
-    else:
-        print("This environment variable has been set, skip")
-    env_path = os.getenv('PATH')
-    path_strings = re.split("[;:]", env_path)
-    for path in path_strings:
-        if path.endswith("depot_tools"):
+    global localLibPath
```

Comment:
is this variable a "BOOLEAN" or a "String" ? It looks like a "String" for me but then why do we use `if localLibPath == False` below?

To make the process simpler, can we make `localLibPath` as a `bool` (maybe the name should be changed too) and still use the `engine_path` as the `localLibPath`. The benefit is that, even if the user has a complete flutter engine codebase, he/she can still execute a `build with local library` using the already built libraries

Reply:
Updated